### PR TITLE
docs: imported new components

### PR DESCRIPTION
**Pull request by andreapiras**
## 📎 Jira ticket
https://transferwise.atlassian.net/browse/DS-161
## ❓ Context  
Imported all components from old docs into new one

## ✅ Checklist

- [ ] All changes are covered by tests
- [ ] The changes are covered in docs
- [ ] The version is bumped according to semver

### CODE REVIEW

File: packages/docs/package.json [hunk 1]
```diff
@@ -36,16 +36,21 @@
...
     "prismjs": "^1.17.1",
     "prop-types": "^15.7.2",
+    "react-docgen": "5.0.0-beta.1",
+    "react-docgen-external-proptypes-handler": "^2.0.0",
+    "react-live": "^2.2.1",
     "refractor": "^2.10.0",
     "rehype": "^9.0.0",
-    "rehype-slug": "^2.0.3"
+    "rehype-slug": "^2.0.3",
+    "shortid": "^2.2.15"
```
Comment (andreapiras) [inline packages/docs/package.json]: Remove

File: packages/docs/static/assets/main.css [hunk 2]
```diff
@@ -140,3 +141,48 @@ pre {
...
+  padding: calc(var(--spacer));
+}
+/*Temp css to remove */
+/* Typography temp */
+.Content h2,
+h3,
+h4 {
+}
+
+.Content h2[id],
```
Comment (andreapiras) [inline packages/docs/static/assets/main.css]: @gusso this is temporary stuff please feel free to override or we can remove it and modify in the next PR

File: packages/components/package.json [hunk 3]
```diff
@@ -31,9 +31,9 @@
     "test:watch": "jest --watch --env=jsdom",
     "lint": "eslint 'src/**/*.js'",
     "lint:fix": "npm run lint -- --fix",
-    "docs:build": "gulp lessCompiler && node scripts/build-docs.js",
-    "docs:deploy": "gh-pages -a -d docs-build",
-    "docs:deploy-for-branch": "deploy-directory-on-branch-to-gh-pages -d docs-build",
+    "docs-old:build": "gulp lessCompiler && node scripts/build-docs.js",
```
Comment (andreapiras) [inline packages/components/package.json]: PLEASE DO NOT DEPLOY THIS

File: packages/docs/package.json [hunk 4]
```diff
@@ -5,11 +5,11 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "dev-new": "next dev",
-    "build": "next build",
-    "docs-new:build": "gh-pages --dotfiles -a -d docs-build",
-    "docs-new:deploy": "deploy-directory-on-branch-to-gh-pages -d docs-build",
-    "docs-new:deploy-for-branch": "deploy-directory-on-branch-to-gh-pages -d docs-build",
+    "dev-new": "node ./docgen.js && next dev",
```
Comment (andreapiras) [inline packages/docs/package.json]: PLEASE DO NOT DEPLOY THIS

File: packages/docs/docgen.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const { writeFileSync } = require('fs');
```
Comment (anttio) [inline packages/docs/docgen.js]: Why the need to import `fs` and then also `writeFileSync`?

Comment (andreapiras) [reply packages/docs/docgen.js]: Yeah it's redundant can use fs.writeFileSync

File: packages/docs/pages/components/Alert.mdx [hunk 6]
```diff
@@ -1,163 +1,272 @@
...
+  </tbody>
+</table>
+
+## Arrow options
+
+<table className="table table-condensed">
+  <thead>
+    <tr>
+      <th>Up-Left</th>
+      <th>Up-Right </th>
```
Comment (anttio) [inline packages/docs/pages/components/Alert.mdx]: Extra whitespace :trollface:

File: packages/docs/pages/components/Button.mdx [hunk 7]
```diff
@@ -1,97 +1,110 @@
...
-Use buttons to let users take actions like starting a transfer or saving the progress and move to
-the next step.
-
-<Button
-  label="Get a free account"
+import { Button, InstructionsList } from '@transferwise/components';
+import { RenderTabs, LiveEditorBlock, GeneratePropsTable } from '../../utils';
+const scope = { Button };
+const code = `<Button
   size="lg"
```
Comment (anttio) [inline packages/docs/pages/components/Button.mdx]: Let's change this to `md` as that's the default size of our buttons.

File: packages/docs/utils/prismsTheme/index.js [hunk 8]
```diff
@@ -0,0 +1,55 @@
+const colors = {
```
Comment (gusso) [inline packages/docs/utils/prismsTheme/index.js]: The filename should be `prismTheme`, no?

File: packages/docs/pages/components/InstructionsList.mdx [hunk 9]
```diff
@@ -1,8 +1,19 @@
...
+  <LiveEditorBlock code={code} scope={{ InstructionsList }} />
+  <GeneratePropsTable componentName="InstructionsList" />
+</>
+#ios
+</RenderTabs>
 
 export const meta = {
   abstractURL: '',
-  name: 'Instructions List',
+  name: 'InstructionsList',
```
[the remark this answers is not in the document]
Comment (andreapiras) [reply packages/docs/pages/components/InstructionsList.mdx]: I have to go through all of them then ?I just put the component name

[the remark this answers is not in the document]
Comment (gusso) [reply packages/docs/pages/components/InstructionsList.mdx]: But I had gone through all of them before and removed the space, so the name of the component is human readable without being related to the implementation. You changed them back now 🙆‍♂️

File: packages/docs/pages/components/Accordion.mdx [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+#web
+<>
+  <LiveEditorBlock code={code} scope={{ Accordion }} />
+  <GeneratePropsTable componentName="Accordion" />
+</>
+#ios
+</RenderTabs>
+
+export const meta = {
+  abstractURL: '',
```
Comment (anttio) [inline packages/docs/pages/components/Accordion.mdx]: I think we could remove the `abstractURL`s, or rename it. (Or is it abstract enough?) /ping @gusso

File: packages/docs/pages/components/JsonSchemaForm.mdx [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+import JsonSchemaForm from '@transferwise/components/src/jsonSchemaForm';
+import code from '../../liveEditorCode/jsonSchemaForm.code';
+
+<RenderTabs>
+  #web
+  <div id="box">
+    <LiveEditorBlock code={code} scope={{ JsonSchemaForm }} />
+    <GeneratePropsTable componentName="JsonSchemaForm" />
+  </div>
+  #ios
```
Comment (anttio) [inline packages/docs/pages/components/JsonSchemaForm.mdx]: Remove the indentation. And surrounding `<div>` could be replaced with fragment?

Comment (andreapiras) [reply packages/docs/pages/components/JsonSchemaForm.mdx]: uhm nope I need the id. Also why you don't want indentation? I found it more readable

File: packages/docs/static/assets/main.css [hunk 12]
```diff
@@ -140,3 +141,48 @@ pre {
...
+  background: #fff;
+  border: 1px solid #f2f5f7;
+}
+
+.live-editor textarea {
+  outline: none;
+}
+/* Tabs */
+.tabs__panel {
+  padding: calc(var(--spacer));
```
Comment (anttio) [inline packages/docs/static/assets/main.css]: No need for the `calc()`.

File: packages/docs/pages/components/Accordion.mdx (hunk 10 above)
Comment (gusso) [reply packages/docs/pages/components/Accordion.mdx]: Yea we have to kill them, I thought I had removed them before. But we can do that as we go through them to write.

File: packages/docs/pages/components/JsonSchemaForm.mdx (hunk 11 above)
Comment (anttio) [reply packages/docs/pages/components/JsonSchemaForm.mdx]: The indentation is different inside the `<RenderTabs>` on this component compared to other components. Not an issue though.

**Pull request merged**